qiskit/optimization/converters/linear_equality_to_penalty.py

```diff
@@ -31,12 +35,14 @@ def __init__(self):
         self._src = None
         self._dst = None
 
-    def encode(self, op: QuadraticProgram, penalty_factor: float = 1e5,
+    def encode(self, op: QuadraticProgram, auto_penalty: bool = True, penalty_factor: float = 1e5,
                name: Optional[str] = None) -> QuadraticProgram:
         """Convert a problem with equality constraints into an unconstrained problem.
 
         Args:
             op: The problem to be solved, that does not contain inequality constraints.
+            auto_penalty: If true, the penalty factor is automatically defined
+                          by `LinearEqualityToPenalty._auto_define_penalty()`.
```

Comment:
I am not sure in the docstrings that we need/should refer to this private method. It should suffice to say the penalty will be automatically computed and people can look at the code easy enough and see it calling that method.